packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -1013,6 +1036,14 @@ class Scaffold extends StatefulWidget {
   ///  * [showModalBottomSheet], which displays a modal bottom sheet.
   final Widget bottomSheet;
 
+  /// Whether the [bottomSheet] is a scrollable widget with [ScrollView.primary]
```

Comment:
I believe this means that the [bottomSheet] has a Scrollable descendant (like a ListView) for which primary is true.

Reply:
Has is probably better, but it also could _be_ one.  We have no way to enforce that.